modules/standard/ChplConfig.chpl

```diff
@@ -129,7 +129,7 @@ module ChplConfig {
   CHPL_MEM = __primitive("get compiler variable", "CHPL_MEM");
 
   /* See :ref:`readme-chplenv.CHPL_MAKE` for more information. */
-  @unstable("'ChplConfig.CHPL_MAKE' is unstable and may be replaced with a different way to access this information in the future");
```

Comment:
Huh... Did this make `CHPL_MAKE` unstable when it wasn't before? If `;` here is the empty statement, then the attribute won't affect `CHPL_MAKE` prior to this PR. Is there a corresponding `test/unstable` test? If not, might be worth adding one.

Reply:
There is not a test of this, but `CHPL_MAKE` was still being marked as unstable before this as well